.rubocop_cocoapods.yml

```diff
@@ -1,67 +1,73 @@
+require:
```

Comment:
this file is pulled in from https://github.com/CocoaPods/shared/blob/master/.rubocop_cocoapods.yml, it should be changed upstream first

Reply:
This file is updated for latest Rubocop, but other repo doesn't do the upgrade yet.

Reply:
it should still be changed there first, then pulled in as other repos update rubocop

Reply:
what's your suggest? let me just push this to `CocoaPods/shared` ?

Reply:
OK, will do

Reply:
https://github.com/CocoaPods/shared/pull/11

Reply:
@segiddins how does it get pulled exactly? I can't find the reference...

Reply:
https://github.com/CocoaPods/Rainforest/blob/1cd6376e526321f11ff9fb9b588d647b4e073a3d/Rakefile#L642-L643